src/models/simulation-magnet.ts

```diff
@@ -30,7 +30,28 @@ export const SimulationMagnet = types
             ? 0
             : self.coilStrength * self.currentStrength;
       },
-
+      get magnetImage(): string {
+        if (self.type === "bar") {
+          switch (self.barStrength) {
+            case 1:
+              return "assets/magnet-bar-3.png";
+            case .6:
```

Comment:
I could have done this in my PR, but would you mind pulling these values into constants somewhere now that we're switching on them?

Reply:
Good idea.  I added constants for these values.

Reply:
Kind of want to eventually rename the image assets with the "weak", "medium", and "strong" identifiers too instead of "1", "2", "3".  Not a big deal, but maybe something to get to eventually.